src/client/src/ui/blotter/components/toolbar/ExcelLauncher.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+import { styled } from 'rt-theme'
+import { PlatformAdapter, withPlatform } from 'rt-components'
+
+const ExcelIcon = () => (
```

Comment:
Maybe excel icon should be in its own file?

Reply:
excel icon moved to different file